features/lorawan/lorastack/mac/LoRaMacCommand.cpp

```diff
@@ -416,8 +426,8 @@ lorawan_status_t LoRaMacCommand::process_mac_commands(uint8_t *payload,
 
 bool LoRaMacCommand::is_sticky_mac_command_pending()
 {
+    //DEAD CODE: mac_cmd_buf_idx_to_repeat is never set
```

Comment:
This is actually a bug in void LoRaMacCommand::parse_mac_commands_to_repeat(). Add 
mac_cmd_buf_idx_to_repeat = cmd_cnt;
to the end of that function.

Reply:
Fixed